strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/configuration/HttpConnectionPoolConfigurationManagementController.java

```diff
@@ -50,7 +50,7 @@
                                          message = "Repository doesn't have remote repository!"),
```

Comment:
(I know this isn't your change, but...) This message is meaningless. It should be `The proxy repository has no associated remote repository`.